lua/common.lua

```diff
@@ -42,3 +41,70 @@ end
 function wesnoth.wml_actions.clear_chat(cfg)
 	wesnoth.interface.clear_chat_messages()
 end
+
+---
+-- Get current game zoom level.
+--
+-- [get_zoom]
+--     variable=zoom
+-- [/get_zoom]
+---
```

Comment:
Just a note here that this might be useful to submit upstream for wesnoth/wesnoth#7423, but that's a separate topic...